src/members/Members.js

```diff
@@ -0,0 +1,20 @@
+import { List } from '@material-ui/core';
+import React from 'react';
+import Member from './member';
+
+const renderMember = (member) => {
+    return (
+        <Member member={member} />
+    )
+}
+
+const Members = ({members}) => {
+
+    return (
+        <List>
+            { members.map((member) => {return renderMember(member)})}
```

Comment:
You could simplify this further.

Remember, when an arrow function (aka anonymous function) has only `return` statement, you can remove curly braces and `return` keyword.

`{ members.map((member) => renderMember(member))`